sstables/compress.hh

```diff
@@ -379,5 +379,11 @@ output_stream<char> make_compressed_file_m_format_output_stream(output_stream<ch
                 sstables::compression* cm,
                 const compression_parameters& cp);
 
-}
-
+input_stream<char> make_uncompressed_file_k_l_format_input_stream(file f,
```

Comment:
Let's use the term checksum_verifying. This isn't about special handling of uncompressed files, it's about special handling of files that have attached checksum data.